src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,116 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.recipe.Ingredient;
+import seedu.address.model.recipe.Name;
+import seedu.address.model.recipe.Recipe;
+import seedu.address.model.recipe.Step;
+
+/**
+ * Displays the contents of a recipe to the result display.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays the details of a recipe identified by the name of "
+            + "the recipe in the recipe list.\n"
+            + "Parameters: name (must be a valid case sensitive name)\n"
+            + "Example: " + COMMAND_WORD + " Mac and cheese";
+
+    public static final String RECIPE_CONTENT = "Name: %s\n\n"
+            + "Ingredients:\n%s\n"
+            + "Steps:\n%s\n"
+            + "Total time: %s\n"
+            + "Servings: %s";
```

Comment:
Just a minor formatting thing, I think Total Time and Servings should be after Name, they're just one-liners and might be more important to the reader.

`CommandResult`'s return value would need to be updated too